rasa/shared/core/trackers.py

```diff
@@ -65,7 +65,7 @@
 from rasa.shared.core.slots import Slot
 
 if TYPE_CHECKING:
-    from typing_extension import TypedDict
+    from typing_extensions import TypedDict
```

Comment:
no idea why this didn't crash before

Reply:
ouch... possibly because of `ignore_missing_imports=True`? we could do do it more granularly, but it's still needed for some 3rd party modules I think.